programs/mango-v4/src/instructions/serum3_place_order.rs

```diff
@@ -258,6 +258,48 @@ pub fn serum3_place_order(
         None
     };
 
+    max_base_qty = if serum_market.is_reduce_only() {
+        require!(
+            load_open_orders_ref(&ctx.accounts.open_orders.as_ref())?.free_slot_bits == u128::MAX,
+            MangoError::SomeError
+        );
+
+        let bank = ctx.accounts.payer_bank.load()?;
+        let token_pos_native = account
+            .token_position(bank.token_index)?
+            .native(&bank)
+            .to_num::<i64>();
```

Comment:
potential rounding issue, -9.1 would be -10

Reply:
hmm would you recommend always flooring?

Reply:
as discussed this whole block is removed